### ISSUE TITLE
Google Meet Grid View is causing issues with Google Meet

### ISSUE THREAD

**Issue by ghost** [Fugiman/google-meet-grid-view#234]
Google Meet Grid View is causing Google Meet to hang - Cannot take yourself off Mute. Cannot present. People can hear you on the meeting but choppy.
It is not internet band width. Started happening thsi week. Prior to this week the extension worked fine.
Had to reboot my laptop for Google Meet to work again. Now it is fine.

**Comment by scottcrevier** [comment]
Same here. Just started a Meet with 2 others, and we all have the extension. Everything locked up, couldn't turn the extension off/on, etc. We all had to disable the extension to get the Meet to work normally. Otherwise, loving it!!

**Comment by annedroiid** [comment]
@svanselous @scottcrevier THis is a duplicate of #225, please follow along there.

**Comment by sarayu1971** [comment]
> Same here. Just started a Meet with 2 others, and we all have the extension. Everything locked up, couldn't turn the extension off/on, etc. We all had to disable the extension to get the Meet to work normally. Otherwise, loving it!!

CHANGE LAYOUT TO SIDE BAR THEN CLICK THE GRID. WORKS GREAT. CAN SEE EVEN MORE THAN 30 STUDENTS. UPDATE EXTENSION TO 1.37. UPDATE CHROME. UPDATE WINDOWS.

**Comment by scottcrevier** [comment]
> > Same here. Just started a Meet with 2 others, and we all have the extension. Everything locked up, couldn't turn the extension off/on, etc. We all had to disable the extension to get the Meet to work normally. Otherwise, loving it!!
> 
> CHANGE LAYOUT TO SIDE BAR THEN CLICK THE GRID. WORKS GREAT. CAN SEE EVEN MORE THAN 30 STUDENTS. UPDATE EXTENSION TO 1.37. UPDATE CHROME. UPDATE WINDOWS.

Yep, did all that. Already had 1.37.0, latest Chrome and Windows 10. Once in the Meet, clicking anything did nothing, so I couldn't change the layout, couldn't even disconnect from the Meet. Had to close the browser tab, then go into extensions to disable it, then go back into the Meet.

**Comment by QWERTYMurdoc** [comment]
Same for our org, only disabling the extension solved it.

**Comment by ruruboxer** [comment]
Yes, we are still seeing the issue. "Kill Pages/unresponsive" happens after a few minutes. Please update this extension. We NEED YOU! :)

**Comment by annedroiid** [comment]
@sarayu1971 @scottcrevier @QWERTYMurdoc Please use the original issue #225 if you have extra information that has not already been covered, do not use duplicate issues. It makes it harder for the author to track everything.

Additionally, adding a comment to effectively say "me too" does nothing but spam the author with emails.

**Comment by ntoupin** [comment]
Made some quick instructions on the workaround:
https://docs.google.com/document/d/15h5JtPj0JyYD5BP03F0TmDABUFC8E33jfaByG0wJS7g

**Comment by sarayu1971** [comment]
> > Same here. Just started a Meet with 2 others, and we all have the extension. Everything locked up, couldn't turn the extension off/on, etc. We all had to disable the extension to get the Meet to work normally. Otherwise, loving it!!
> 
> CHANGE LAYOUT TO SIDE BAR THEN CLICK THE GRID. WORKS GREAT. CAN SEE EVEN MORE THAN 30 STUDENTS. UPDATE EXTENSION TO 1.37. UPDATE CHROME. UPDATE WINDOWS.

I HAVE DONE THIS AND ALL MY  120 TEACHERS ARE ABLE TO SEE AL THE CHILDREN IN CLASS.

**Comment by sarayu1971** [comment]
> @sarayu1971 @scottcrevier @QWERTYMurdoc Please use the original issue #225 if you have extra information that has not already been covered, do not use duplicate issues. It makes it harder for the author to track everything.
> 
> Additionally, adding a comment to effectively say "me too" does nothing but spam the author with emails.

Hello madam..Take it or leave it.. I have done what I need to and works great for me and all my staff..all the best to you

**Comment by sarayu1971** [comment]
Hello friends. I have worked around the issue with some help from Github .It has worked well for my staff of 120 teachers and school of 2000 plus kids.

> > > Same here. Just started a Meet with 2 others, and we all have the extension. Everything locked up, couldn't turn the extension off/on, etc. We all had to disable the extension to get the Meet to work normally. Otherwise, loving it!!
> > 
> > 
> > CHANGE LAYOUT TO SIDE BAR THEN CLICK THE GRID. WORKS GREAT. CAN SEE EVEN MORE THAN 30 STUDENTS. UPDATE EXTENSION TO 1.37. UPDATE CHROME. UPDATE WINDOWS.
> 
> Yep, did all that. Already had 1.37.0, latest Chrome and Windows 10. Once in the Meet, clicking anything did nothing, so I couldn't change the layout, couldn't even disconnect from the Meet. Had to close the browser tab, then go into extensions to disable it, then go back into the Meet.

Hi..When you open meet first change the layout to sidebar then untick the grid on the meet

**Comment by pablogirado** [comment]
Same error.

**Comment by ajgankema** [comment]
Also with the Google Meet layout set to sidebar unpredictable problems are caused by Grid View: Freezes, Page not responding, no sound etc.
Sorry to say but version 1.37.0 of Grid View is unreliable at this moment. 
But at the University of Groningen wel love Grid View and please continue the good work!

**Comment by pablogirado** [comment]
Try with workaround: https://docs.google.com/document/d/15h5JtPj0JyYD5BP03F0TmDABUFC8E33jfaByG0wJS7g/edit

but it worked for me with the Spotlight view, not with the Sidebar view....

![WhatsApp Image 2020-06-22 at 20 24 51](https://user-images.githubusercontent.com/65799214/85410025-f289be00-b53c-11ea-9a3d-679438a71ffd.jpeg)
